frappe/desk/doctype/form_tour_step/form_tour_step.json

```diff
@@ -0,0 +1,85 @@
+{
+ "actions": [],
+ "creation": "2021-05-21 23:05:45.342114",
+ "doctype": "DocType",
+ "engine": "InnoDB",
+ "field_order": [
+  "field",
+  "title",
+  "description",
+  "column_break_2",
+  "position",
+  "fieldname",
+  "label",
+  "condition"
+ ],
+ "fields": [
+  {
+   "fieldname": "title",
+   "fieldtype": "Data",
+   "in_list_view": 1,
+   "label": "Title",
+   "reqd": 1
+  },
+  {
+   "columns": 4,
+   "fieldname": "description",
+   "fieldtype": "HTML Editor",
+   "in_list_view": 1,
+   "label": "Description",
+   "reqd": 1
+  },
+  {
+   "fieldname": "field",
+   "fieldtype": "Link",
+   "label": "Field",
+   "options": "DocField",
+   "reqd": 1
+  },
+  {
+   "fetch_from": "field.fieldname",
+   "fieldname": "fieldname",
+   "fieldtype": "Data",
+   "hidden": 1,
+   "label": "Fieldname",
+   "read_only": 1
+  },
+  {
+   "fetch_from": "field.label",
+   "fieldname": "label",
+   "fieldtype": "Data",
+   "in_list_view": 1,
+   "label": "Label",
+   "read_only": 1
+  },
+  {
+   "fieldname": "column_break_2",
+   "fieldtype": "Column Break"
+  },
+  {
+   "default": "Bottom",
+   "fieldname": "position",
+   "fieldtype": "Select",
+   "label": "Position",
+   "options": "Left\nLeft Center\nLeft Bottom\nTop\nTop Center\nTop Right\nRight\nRight Center\nRight Bottom\nBottom\nBottom Center\nBottom Right\nMid Center"
+  },
+  {
+   "fieldname": "condition",
+   "fieldtype": "Code",
+   "label": "Next Condition",
```

Comment:
what is this supposed to do? condition for the next step?
```suggestion
   "label": "Next Step Condition",
```
and field name can also be `next_step_condition`